content/nginx/deployment-guides/amazon-web-services/ec2-instances-for-nginx.md

```diff
@@ -1,186 +1,144 @@
 
-6. In the **Step 4: Add Storage** window, leave the defaults unchanged. Click the <span style="background-color:#cccccc; font-weight:bold;"> Next: Add Tags </span> button.
+1. In the **Application and OS Images (Amazon Machine Image)** section click the image of the Linux distribution of your choice.
 
-   <a href="/nginx/images/aws-nlb-instance-add-storage.png"><img src="/nginx/images/aws-nlb-instance-add-storage.png" alt="" width="1024" height="428" class="aligncenter size-full wp-image-54833" style="border:2px solid #666666; padding:2px; margin:2px;" /></a>
+   {{< img src="/img/aws/aws-nlb-instance-choose-ami.png" alt="Screenshot of AMI section on the EC2 Launch Instance page">}}
 
-7. In the **Step 5: Add Tags** window, click the <span style="background-color:#cccccc; font-weight:bold;"> Add Tag </span> button. Type <span style="color:#666666; font-weight:bolder;">Name</span> in the **Key** field, and in the **Value** field type the instance name (the screenshot shows the result). This name is what will appear in the **Name** column of the summary table on the **Instances** tab of the EC2 dashboard (see the screenshot in Step 12, which shows one instance).
+1. In the **Instance Type** section, use the dropdown to choose an appropriate instance type. The screenshot shows the **t2.micro** instance type selected by default. This type is sufficient for demo purposes.
 
-   If you are following these instructions as directed by an NGINX deployment guide, the **Creating EC2 Instances and Installing the NGINX Software** section of the deployment guide specifies the instance names to use.
+   {{<note >}}At the time of publication, AWS offered 750 hours of free usage each month for this instance type. This applies during your first year with an AWS account. Keep in mind, though, that several NGINX instances running all day will use the free 750 hours up quickly. For example, 6 instances will use them in just over 5 days. If you use 8 instances, you'll hit the limit in under 4 days.{{</note>}}
 
-   Click the <span style="background-color:#cccccc; font-weight:bold;"> Next: Configure Security Group </span> button to continue to the next step.
+   {{< img src="/img/aws/aws-nlb-instance-choose-type.png" alt="Screenshot of Instance Type on the EC2 Launch Instance page">}}
 
-   <a href="/nginx/images/aws-instance-add-tags-name.png"><img src="/nginx/images/aws-instance-add-tags-name.png" alt="" width="1024" height="483" class="aligncenter size-full wp-image-56334" style="border:2px solid #666666; padding:2px; margin:2px;" /></a>
+1. In the **Key pair (login)** section, you can select an existing key pair, or create a new one. If you click on "Create new key pair," a window will pop up allowing you to download a key pair to your specifications.
```

Comment:
```suggestion
1. In the **Key pair (login)** section, select an existing key pair or create a new one. If you choose **Create new key pair**, a window appears, allowing you to download the key pair.
```

Reply:
style guide edits:

- Use "select" instead of "click." 
- Use present tense predominately.